schemas/api_schema.js

```diff
@@ -174,6 +175,35 @@ type Mutation {
                     where: { category },
                     limit: count
                 });
+            },
+            SearchSessions: (parent, { keywords }, { db }, info) => {
+                const Op = db.Sequelize.Op;
+                var words = keywords.split(' ');
+                var sqlWords = words.map(function (word) {
```

Comment:
use arrow functions and format strings:
`(word) => ({ [Op.like]: `%${word}%` })`

don't forget the parentheses to explicitly tell the parser that this is an object and not a block